<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Regression" duration-ms="108879" started-at="2023-04-10T18:41:56Z" finished-at="2023-04-10T18:43:45Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="108879" started-at="2023-04-10T18:41:56Z" finished-at="2023-04-10T18:43:45Z">
      <class name="tests.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:tests.LoginTest@1df8da7a]" name="setUp" is-config="true" duration-ms="631" started-at="2023-04-10T11:41:56Z" finished-at="2023-04-10T11:41:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setDriver(java.lang.reflect.Method)[pri:0, instance:tests.LoginTest@1df8da7a]" name="setDriver" is-config="true" duration-ms="6346" started-at="2023-04-10T11:41:56Z" finished-at="2023-04-10T11:42:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.LoginTest.loginTest() throws java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setDriver -->
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:tests.LoginTest@1df8da7a]" name="loginTest" duration-ms="22847" started-at="2023-04-10T11:42:03Z" finished-at="2023-04-10T11:42:26Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed to logout expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed to logout expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at tests.LoginTest.loginTest(LoginTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="removeDriver()[pri:0, instance:tests.LoginTest@1df8da7a]" name="removeDriver" is-config="true" duration-ms="888" started-at="2023-04-10T11:42:27Z" finished-at="2023-04-10T11:42:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeDriver -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:tests.LoginTest@1df8da7a]" name="tearDown" is-config="true" duration-ms="1343" started-at="2023-04-10T11:43:45Z" finished-at="2023-04-10T11:43:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.UserMenuTest">
        <test-method status="PASS" signature="setDriver(java.lang.reflect.Method)[pri:0, instance:tests.UserMenuTest@23941fb4]" name="setDriver" is-config="true" duration-ms="2762" started-at="2023-04-10T11:42:28Z" finished-at="2023-04-10T11:42:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.UserMenuTest.userMenu_TC06() throws java.io.IOException,java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setDriver -->
        <test-method status="PASS" signature="userMenu_TC06()[pri:0, instance:tests.UserMenuTest@23941fb4]" name="userMenu_TC06" duration-ms="74169" started-at="2023-04-10T11:42:30Z" finished-at="2023-04-10T11:43:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userMenu_TC06 -->
        <test-method status="PASS" signature="removeDriver()[pri:0, instance:tests.UserMenuTest@23941fb4]" name="removeDriver" is-config="true" duration-ms="717" started-at="2023-04-10T11:43:44Z" finished-at="2023-04-10T11:43:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeDriver -->
      </class> <!-- tests.UserMenuTest -->
    </test> <!-- Test -->
  </suite> <!-- Regression -->
</testng-results>
